zulipterminal/ui_tools/views.py

```diff
@@ -1165,6 +1165,10 @@ def _fetch_user_data(controller: Any, user_id: int) -> Dict[str, Any]:
             )
             display_data["Local time"] = local_time[11:]
 
+        if data["custom_profile_data"]:
+            for field in data["custom_profile_data"]:
+                display_data[field] = data["custom_profile_data"][field]
```

Comment:
I know you mentioned that the last-online field is useful to have last, this place isn't the best to accomplish this.

This function is concerned with preparing the TidiedUserInfo for output, so we don't need to enforce the ordering in this function. A more general response would be to return the regular data and this data as a tuple. A simple way to group the custom profile data would then be to put it into a group like we do with the actions/links/reactions in the message info popup.